l10n_ec_ote/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+{
+    'name': 'OTE for Ecuador',
+    'depends': ['base', 'sales_team','base_address_city'],
+    'category': 'Localization',
+    'version': '15.0.0.1.0',
```

Comment:
Initial version must be 15.0.1.0.0 according [OCA guideslines ](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#12version-numbers)
```suggestion
    'version': '15.0.1.0.0',
```